src/shadow-embed.js

```diff
@@ -61,7 +62,10 @@ export function createShadowRoot(hostElement) {
   // Native support.
   const shadowDomSupported = getShadowDomSupportedVersion();
   if (shadowDomSupported == ShadowDomVersion.V1) {
-    return hostElement.attachShadow({mode: 'open'});
+    // NOTE(torch2424) need to have an id on the shadow root no matter what
+    const shadowRoot = hostElement.attachShadow({mode: 'open'});
```

Comment:
Oh my apologies, yes I meant `createShdowRootPolyfill()`, the github code hide thing didn't show the function call. 

Yes you are correct, I tricked myself, as chrome was also falling into ShadowCSS not being supported even though it is, since I have the dangling todo in `isShadowCSSSupported()`, removing....